### PR TITLE
Various improvements in Contribute section

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

**Comment by lex111** [comment]
Any news?

File: content/en/docs/contribute/_index.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ To get involved with documentation:
 
 ## Contributions best practices
 
-- Do write clear and meaningful GIT commit messages.
-- Make sure to include _Github Special Keywords_ which references the issue and automatically closes the issue when PR is merged.
-- When you make a small change to a PR like fixing a typo, any style change, or changing grammar. Make sure you squash your commits so that you dont get a large number of commits for a relatively small change.
+- Do write clear and meaningful Git commit messages.
+- Make sure to include _GitHub Special Keywords_ which references the issue and automatically closes the issue when PR is merged.
+- When you make a small change to a PR like fixing a typo, any style change, or changing grammar. Make sure you squash your commits so that you don't get a large number of commits for a relatively small change.
 - Make sure you include a nice PR description depicting the code you have changes, why to change a following piece of code and ensuring there is sufficient information for the reviewer to understand your PR.
```
Comment (prankul88) [inline content/en/docs/contribute/_index.md]: Slight Nits:
> Make sure you include a nice PR description depicting the code you have changes

Make sure you include a nice PR description depicting the code you have changed

**Comment by prankul88** [comment]
LGTM otherwise. Thanks

**Comment by jimangel** [comment]
@lex111 will you please rebase? Thanks!

**Comment by jimangel** [comment]
@lex111 this has been pending for 2 weeks with the need to rebase. Please reopen if you'd like to continue working on it after rebasing.

/close

**Pull request closed without merging**